pkg/model/lattice/accesslogsubscription.go

```diff
@@ -0,0 +1,52 @@
+package lattice
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-application-networking-k8s/pkg/model/core"
+)
+
+type SourceType string
+
+const (
+	ServiceNetworkSourceType SourceType = "ServiceNetwork"
+	ServiceSourceType        SourceType = "Service"
+)
+
+type AccessLogSubscription struct {
+	core.ResourceMeta `json:"-"`
+	Spec              AccessLogSubscriptionSpec    `json:"spec"`
+	Status            *AccessLogSubscriptionStatus `json:"status,omitempty"`
+}
+
+type AccessLogSubscriptionSpec struct {
+	SourceType     SourceType
+	SourceName     string
```

Comment:
More cleaner to change to `SourceLatticeResourceType`, `SourceLatticeResourceName` ?

Reply:
It's talking about AccessLogSubscription, which is already a Lattice resource